test/EndToEndTests/Tests/Client/AsynchronousTests/AsynchronousQueryTests.cs

```diff
@@ -308,7 +307,7 @@ public void CancelBatchRequestTest()
             {
                 if (args.RequestMessage is HttpWebRequestMessage)
                 {
-                    servicePoint = ((HttpWebRequestMessage)args.RequestMessage).HttpWebRequest.ServicePoint;
+                   

```

Comment:
If this `if` statement does not do anything, it could probably be removed... and the handler itself may be unnecessary?

Reply:
this if statement is inside a test that has been cancelled out. I do not want to delete it because it is linked to a GitHub issue meaning it can be worked on later.

Reply:
Added a work item to check why the test was skipped and fix.